agents.yaml

```
@@ -388,59 +397,54 @@ data:
       max_bytes 500000
       max_lines 1000
     </match>
+  # Add 0. prefix to make sure this is configured before output.
+  0.pods.input.conf: |-
+    # This configuration file for Fluentd is used
+    # to watch changes to Kubernetes pod log files live in the
+    # directory /var/log/pods/NAMESPACE_NAME_UID. The file name
+    # is used to capture the pod namespace, name and uid. These
+    # logs are then submitted to Google Cloud Logging with a
+    # local_resource_id 'k8s_pod.<NAMESPACE_NAME>.<POD_NAME>'
+    # which assumes the installation of the cloud-logging plug-in.
+    <source>
+      @type tail
+      {{ if .SystemOnlyLogging }}
+      path /var/log/pods/kube-system_*/*,/var/log/pods/istio-system_*/*,/var/log/pods/knative-serving_*/*,/var/log/pods/gke-system_*/*,/var/log/pods/config-management-system_*/*,
+      {{ else }}
+       path /var/log/pods/*/*.log
```

Use path in `else` condition would enable us to get all the logs, not only system logs.

nit: space is not aligned, so I added a suggestion here.
 
```suggestion
      path /var/log/pods/*/*.log
```